src/pal/src/exception/seh-unwind.cpp

```diff
@@ -72,8 +72,19 @@ static void WinContextToUnwindCursor(CONTEXT *winContext, unw_cursor_t *cursor)
     unw_set_reg(cursor, UNW_X86_64_R13, winContext->R13);
     unw_set_reg(cursor, UNW_X86_64_R14, winContext->R14);
     unw_set_reg(cursor, UNW_X86_64_R15, winContext->R15);
-#else
-#error unsupported architecture
+#elif defined(_ARM_)
+    unw_set_reg(cursor, UNW_REG_IP, winContext->Pc);
+    unw_set_reg(cursor, UNW_REG_SP, winContext->Sp);
+    unw_set_reg(cursor, UNW_ARM_R4, winContext->R4);
+    unw_set_reg(cursor, UNW_ARM_R5, winContext->R5);
+    unw_set_reg(cursor, UNW_ARM_R6, winContext->R6);
+    unw_set_reg(cursor, UNW_ARM_R7, winContext->R7);
+    unw_set_reg(cursor, UNW_ARM_R8, winContext->R8);
+    unw_set_reg(cursor, UNW_ARM_R9, winContext->R9);
+    unw_set_reg(cursor, UNW_ARM_R10, winContext->R10);
+    unw_set_reg(cursor, UNW_ARM_R11, winContext->R11);
+    //unw_set_reg(cursor, UNW_ARM_R12, winContext->R12);
+    unw_set_reg(cursor, UNW_ARM_R14, winContext->Lr);
```

Comment:
Please hoist LR up to the other special register handling ($pc, $sp) should be a bit more readable.